src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -164,3 +173,7 @@ class MarathonScheduler(
     */
   def getLocalRegion: Option[Region] = localFaultDomain.map(_.region)
 }
+
+object MarathonScheduler {
+  val MAX_STATUS_MESSAGE_LENGTH = 10000
```

Comment:
Assuming default 1Mb znode size, and let's say we could use half of that for the status message, we would be able to store `0.5 * 1024 * 1024` bytes. You could also, check for `status.getMessage.getBytes.length` then.

Reply:
Yeah, but you'd have to take into account that we Base64 encode the whole TaskStatus when embedding it into the JSON of the Instance... So, we could probably store a longer message, but tbh, no reasonable error message should be longer than 10k characters.

We do log the full message before truncating it in any case.